xdebug_com.c

```diff
@@ -73,7 +97,16 @@ int xdebug_create_socket(const char *hostname, int dport TSRMLS_DC)
 	struct pollfd              ufds[1];
 	long                       optval = 1;
 #endif
-	
+
+	if (!strncmp(hostname, "unix://", strlen("unix://"))) {
```

Comment:
Yes, but udg:// doesn't make a lot of sense for xdebug.  If it did, you'd support UDP already. :p